### ISSUE TITLE
Please add backbone selection to compute_lrmsd_pdb2sql()

### ISSUE THREAD

**Issue by DarioMarzella** [DeepRank/pdb2sql#55]
**Is your feature request related to a problem? Please describe.**
I would be able to calculate l-RMSD on different atoms of the backbone: only CA, or [CA, C, O, N] or [CA, CB, C, O, N], etc.

**Describe the solution you'd like**
Allowing the user to give a list of atoms to be used for l-RMSD calculation as an optional parameter of compute_lrmsd_pdb2sql

**Describe alternatives you've considered**
This might be appliable to irmsd as well.

**Comment by NicoRenaud** [comment]
The easiest solution is to pass the selection keyword of `pdb2sql.get()` down to the 
`compute_lrmsd_pdb2sql()` I'm not sure about the fast methods

**Comment by NicoRenaud** [comment]
first attempt in `issue55` branch. we ca use 
```python
s.compute_lrmsd_pdb2sql(name=['CA','O'])
s.compute_lrmsd_fast(name=['CA','O'])
```

if the `name` kwarg is not specified it defaults to [C, CA, N, O]

**Comment by NicoRenaud** [comment]
need to be extended to irmsd (?) and cleaned up

**Comment by CunliangGeng** [comment]
Solved in PR #59
